lib/RunTests.js

```diff
@@ -67,13 +70,13 @@ function makeProgressLogger(report /*: typeof Report.Report */) {
     },
     clearLine() {
       items.length = 0;
-      if (!Report.isMachineReadable(report)) {
+      if (!Report.isMachineReadable(report) && !preserveOutput) {
```

Comment:
I think we still want to allow clearing the current line?

Reply:
yes, you are right